README.md

```diff
@@ -17,15 +17,15 @@ This addon provides a service to schedule work in different paint phases of rend
 The `afterFirstRoutePaint` queue can be used to schedule work after the route is first painted. This is useful for scenarios like rendering content outside viewport, rendering non critical content etc.
 
 ```javascript
-this.get('scheduler').on('afterFirstRoutePaint', () => {
```

Comment:
YAAASS `'on'` made this seem evented.

Reply:
Yes, but the interface felt more natural.